docs/reference/ml/ml-shared.asciidoc

```diff
@@ -863,9 +863,10 @@ example, `1575402236000 `.
 end::model-snapshot-id[]
 
 tag::model-snapshot-retention-days[]
-The time in days that model snapshots are retained for the job. Older snapshots
-are deleted. The default value is `1`, which means snapshots are retained for
-one day (twenty-four hours).
+Advanced configuration option. Denotes the period for which model snapshots
```

Comment:
If this property is still a number that represents days, I think it's worth keeping that in the definition.  For example:
```suggestion
Advanced configuration option. The period of time (in days) that model snapshots are retained.
```

Reply:
Yes that's better thanks